crates/typst/src/layout/place.rs

```diff
@@ -77,9 +78,13 @@ pub struct PlaceElem {
     ///   place(center, dx: amount - 32pt, dy: amount)[A]
     /// }
     /// ```
+    ///
+    /// This currently does not affect layout, even if `float` is `{true}`.
```

Comment:
The wording "does not affect layout" is generally not great imo (if it does not affect layout, what does it do?) It's also a bit of a problem for `move`, `scale`, etc, but here it feels extra confusing. also, what is the special connection to `float`? why would float change this?

Reply:
If `float` is true, then the `place` element is visible to the layout algorithm, but the layout algorithm ignores the `dx` and `dy` parameters (i.e. these are treated as if the inner content were wrapped in a `move` element).

Reply:
Yeah, okay, I see. Still, maybe there is some way to explain this more clearly. We can also have more than one sentence if necessary.

Reply:
What do you think about the new wording in 0f46284239c6f22a8d84ef98597ee27d98c1728f?